aries-site/src/pages/components/pagination.mdx

```diff
@@ -45,46 +72,9 @@ Below are some accessibility requirements that ensure pagination is accessible f
 If your team is developing with Grommet, these considerations are already handled, so no further action is required from your team.
 
 1. Wrap pagination links in a `<nav>` element.
-1. Provide an `aria-label="pagination navigaton"` and `role="navigation"`.
-1. Individual page buttons should have an `aria-label` that reads "Go to page X", where X is the page that will be navigated to upon selection.
-1. The active page button should have `aria-current="true"`.
-1. The next/previous arrow buttons should have `aria-label` that reads "Go to next page" and "Go to previous page", respectively.
-
-For more detailed documentation on these requirements, you can read more about [accessible pagination](https://www.a11ymatters.com/pattern/pagination/).
-
-## Variants
-
-In addition to [DataTable](/components/datatable#paginated) and [List](/templates/lists#paginated), here is another way pagination can be implemented.
-
-### Paginated table
+2. Provide an `aria-label="pagination navigaton"` and `role="navigation"`.
```

Comment:
Just fyi -- in markdown you can leave as "1." and it automatically renders the correct number based on what number it is in a list. Makes it easier in future if we ever add/remove items.